exodus_analyze.py

```diff
@@ -105,6 +107,13 @@ def main():
         default=None,
         help='comma-separated ids of trackers to ignore'
     )
+    parser.add_argument(
+        '--no-exit-counter',
```

Comment:
Could it be possible to have a shorter option name? I find `--no-exit-counter` quite long, especially with 2 dashes in the option name.

Reply:
I use '-e'. Is it OK?